RELEASENOTES/v2.0.0.rc-2.MD

```diff
@@ -0,0 +1,68 @@
+    - Use a new "completion" command to generate the corresponding completion script for their shells. 
+  
+  - Updated libraries for Notation for
+    - Improved error messaging, documentation, dependencies and bug fixes
+
+    
+*This is the second supported release from the Notary v2 community. Refer prior (RC-1) release description [here](https://github.com/notaryproject/roadmap/blob/main/RELEASENOTES/v2.0.0.rc-1.MD) and release definitions [here](https://github.com/notaryproject/notation/blob/main/RELEASE_MANAGEMENT.md)
+
+## List of feature/capabilities added with this release
+For detailed notes on each of the individual repositories (NotaryProject, Notation (for reference CLI implementation) and Notation libraries) users can refer to the individual release notes. A summary view is provided in this document.
+
+### Notation CLI Client
+Notation is a standalone CLI client based on Notary v2 specs. It supports signing, verifying, and storing signatures in registries supporting signatures as references to images.
+
+In this release of Notation command, the following CLI commands are supported. 
+  - **sign**        Sign OCI artifacts
+  - **verify**      Verify OCI artifacts
+  - **certificate** Manage certificates in trust store for verifying 
+  - **key**         Manage keys used for signing
+  - **list**        List signatures of the signed artifact
+  - **login**       Login to registry
+  - **logout**      Log out from the logged in registries
+  - **plugin**      Manage plugins  
+  - **version**     Show the notation version information
+  - **help**        Help about any command
+  *The following two new commands were added in this RC-2 release*
+  - **inspect**     Inspect all signatures associated with the signed artifact 
+  - **completion**  Generate the autocompletion script for the specified shell
+
+As part of this release, users have to create a trust policy configuration file before verifying arifacts. 
+
+To start using Notation Refer: https://notaryproject.dev/docs/quickstart/ 
+
+### Notation Library 
+Written in the GO language the Library is divided up into two separate repos - [Notation-go](https://github.com/notaryproject/notation-go) and [Notation-core-go](https://github.com/notaryproject/notation-core-go). Updated Notation-go for
+- Adding optional meta-data at time of signing images
+- Added  check for plugin versioning and compatability
+
+## List of fixed issues
+Refer individual release notes.
+
+## Roadmap Guidance
+*This is not an exhaustive list. Users should refer to the Notary v2 roadmap at https://github.com/notaryproject/roadmap#readme for seeing the plan for subsequent releases and refer to individual subprojects (repositories) releases notes for details*
+
+1. Revocation support via Notation client will come in a future release. Spec update for revocation support is planned for the next release candidate
+2. CLI specs for managing trust policy is planned for the next release candidate
+
```

Comment:
Could you add the follows based on our discussions?
- Sign/verify offline artifacts
- UX improvement for notation plugin